bindings/ruby/Cargo.toml

```diff
@@ -18,7 +18,7 @@
 [package]
 name = "opendal-ruby"
 publish = false
-version = "0.1.11"
+version = "0.1.12"
```

Comment:
We don't need to update it in this PR; the release manager will take care of it during the release.

Reply:
Thanks for future release manager!!